tests/gis_tests/test_utils.py

```diff
@@ -0,0 +1,53 @@
+import unittest
```

Comment:
I'd call this something like `test_gis_tests_utils.py` to clear confusion that it's not testing any `utils.py` in `contrib.gis`.